### PR TITLE
add superAdmin and superAdminColumn

### PR DESCRIPTION
**Pull request by saeideng**

### CODE REVIEW

**Comment by saeideng** [comment]
fixed CS

**Comment by saeideng** [comment]
ping
other suggest:
move code to new Component and use them in this  adapter 
maybe we can use component code in other place  in our app

**Comment by dereuromark** [comment]
Well, I am not against this change, so far I always did that in project level with 2-3 lines though:
```php
...
if ($this->AuthUser->hasRole(ROLE_SUPERADMIN)) {
	$this->Auth->allow();
}
...
```

**Comment by dereuromark** [comment]
If you want to add this there should be tests IMO.

**Comment by saeideng** [comment]
these changes useful when, we use permission system for create  menu or other usage 
we can use manual code but use of superAdmin option is very simple :D

maybe move codes to component is better way to use that in other place

**Comment by dereuromark** [comment]
@bravo-kernel any objections?

File: docs/README.md [hunk 1]
```diff
@@ -211,6 +211,8 @@ rolesTablePlugin|string|Name of the plugin for the roles table, if any.
 pivotTablePlugin|string|Name of the plugin for the pivot table, if any.
 multiRole|bool|True will enable multi-role/HABTM authorization (requires a valid join table)
 superAdminRole|int|Id of the super admin role. Users with this role will have access to ALL resources.
+superAdmin|int or string|Id/name of the super admin. Users with this id/name will have access to ALL resources.null/0/"0" take disable it
```
Comment (bravo-kernel) [inline docs/README.md]: Change `Users with` to `User with` ?

**Comment by bravo-kernel** [comment]
Please excuse me if I am misinterpreting this but to be honest I don't really understand the rational behind this change. What's the point of adding support for a single user who is given `SuperAdmin` power when thats's basically the same as giving only one user the `superAdminRole` role?

**Comment by saeideng** [comment]
@bravo-kernel  no
we not have just a super admin
We can create group of admins and use one of that groups as super admin
we can create group with idColumn

File: docs/README.md (hunk 1 above)
Comment (saeideng) [inline docs/README.md]: if we use group of user maybe Users is correct 
But i can change it

**Comment by bravo-kernel** [comment]
@saeideng aha, I understand now. @dereuromark no objections from me. 

@both I feel the list op options is becoming (too) long? Would it be possible to refactor, pointer welcome.

**Comment by dereuromark** [comment]
Would still be good to add some tests for this before we release it.

**Comment by saeideng** [comment]
Ok
I add some tests as soon

**Pull request merged**